### DISCUSSION TITLE
Database error during installation

### DISCUSSION THREAD

**Discussion by asuri0n** [amidaware/tacticalrmm#1350] [General]
I tried to install TRMM on a fresh debian 11 install (OVH VPS)
I followed the traditional installation guide

I have error during script installation before asking the RMM website credentials : 
![image](https://user-images.githubusercontent.com/15729117/201478938-0008b208-3fbe-463b-b2aa-61881246f72c.png)

Do you have any tips ? :) Thanks

I tried [this solution](https://github.com/amidaware/tacticalrmm/discussions/1209)

**Comment by silversword411** [comment]
Are you having login problems?

Start with a redeploy 1st

https://docs.tacticalrmm.com/troubleshooting/#problems-after-new-server-install

Permission error looks unusual

**Reply by asuri0n** [reply]
Same with new redeploy

**Comment by wh1te909** [comment]
There would have been other errors prior to getting here, you need to check those first to see why database didn't install. Reinstall and keep a log of the install script so we can see where it's failing.

**Reply by asuri0n** [reply]
What part do you want ? 
Here is the postgres part : 
![image](https://user-images.githubusercontent.com/15729117/201600799-b3540c09-0274-42c0-a99b-b976f10d36b9.png)

do you want the entire log ?

**Reply by asuri0n** [reply]
All seems good, right ?
![image](https://user-images.githubusercontent.com/15729117/201603372-bb296ba7-0df4-4daa-9d1a-248ce9659a96.png)

**Reply by wh1te909** [reply]
"database already exists" from your screenshot. you can't run the install script again if it failed, need to wipe and start with fresh vm. do that and if it still errors please email the full log to `help@amidaware.com`
